docs/get-started/traces.md

```diff
@@ -0,0 +1,59 @@
+A **span** is the building block of a trace. You might also think of spans as logs with extra functionality — a single row in our live view.
+
+Spans let you add context to your logs and measure code execution time. Multiple spans combine to form a trace,
+providing a complete picture of an operation's journey through your system.
```

Comment:
Should we highlight this in some sort of admonition box?

https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types

Reply:
I didn't think it made sense for both paragraphs but I added one - does help break up the content, for sure.